optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/common/jaxrs/OptaShiftObjectMapperResolver.java

```diff
@@ -34,7 +31,10 @@
 
     public OptaShiftObjectMapperResolver() {
         objectMapper = new ObjectMapper()
-                .registerModule(new JavaTimeModule());
+                .registerModule(new JavaTimeModule())
+                .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+                // This converts the server side offset to GMT, is that what we want?
+                .setDateFormat(new ISO8601DateFormat());
```

Comment:
No, for example for OffsetDateTime, we want to have the same as it's toString(), which is an ISO standard that includes the offset. For example: "2007-12-03T10:15:30+01:00"

From OffsetDateTime's javadoc:

```
    /**
     * Outputs this date-time as a {@code String}, such as {@code 2007-12-03T10:15:30+01:00}.
     * <p>
     * The output will be one of the following ISO-8601 formats:
     * <ul>
     * <li>{@code uuuu-MM-dd'T'HH:mmXXXXX}</li>
     * <li>{@code uuuu-MM-dd'T'HH:mm:ssXXXXX}</li>
     * <li>{@code uuuu-MM-dd'T'HH:mm:ss.SSSXXXXX}</li>
     * <li>{@code uuuu-MM-dd'T'HH:mm:ss.SSSSSSXXXXX}</li>
     * <li>{@code uuuu-MM-dd'T'HH:mm:ss.SSSSSSSSSXXXXX}</li>
     * </ul>
     * The format used will be the shortest that outputs the full value of
     * the time where the omitted parts are implied to be zero.
     *
     * @return a string representation of this date-time, not null
     */
    @Override
    public String toString() {
        return dateTime.toString() + offset.toString();
    }
```
The abbreviation would be nice to have, but not required. The offset inclusion is required (otherwise there is data loss).
I suspect the jackson-java8Time module docs will document the best way to achieve this.